Code/C++/check_if_bipartite_graph.cpp

```diff
@@ -0,0 +1,103 @@
+/*
+Check if a graph is bipartite graph.
+
+A Bipartite graph is a graph whose vertcies can be divided into independent sets such that for every edge(u,v) u and v belong to different set i.e all edges should be from one set to another.
+A bipartite graph is possible if the graph coloring is possible using two colors such that vertices in a set are colored with same color.
+*/
```

Comment:
Merge these 2 into a single multiline comment

Reply:
@raksha009 I have done the changes. Please review it.